docs/user-guide/paragraph-section.md

```diff
@@ -0,0 +1,46 @@
+# Paragraph Section
+**********************
+
+The Paragraph Section allows to insert a textual content to the story. The story editor can also click on the <img src="../img/button/++++.jpg" class="ms-docbutton"/> button to add additional contents to this section (like media, other parafraphs or embed third party contens). It is possible to chose between:
+
+<img src="../img/paragraph-section/add-paragraph.jpg" class="ms-docimage"/>
+
+* **Text Content** <img src="../img/button/add-text-content.jpg" class="ms-docbutton"/> to add another text content just below the current one
+
+* **Media Content** <img src="../img/button/add-media-content.jpg" class="ms-docbutton"/> to open the Media Editor to add an image, a map or a video.
+
+* **Web Page Content** <img src="../img/button/web-page-button.jpg" class="ms-docbutton"/> to add an external web page 
+
+## Text Content
+
+By default, as soon as a Paragraph is added, an empty text content is already present as a placeholder:
+
+<img src="../img/paragraph-section/paragraph-section.jpg" class="ms-docimage"/>
+
+With a click on the text area, it is possible to write the text and customize it through the [Text Editor Toolbar](text-editor-toolbar.md).  A possible result of adding and formatting the text can be the following: 
+
+<img src="../img/paragraph-section/p-text-content.jpg" class="ms-docimage"/>
+
+The user can delete the text content with the Remove button  <img src="../img/button/remove2.jpg" class="ms-docbutton"/>.
+
+## Media Content
+
+Adding a media content, the [Media Editor Window](media-editor-window.md) opens to allow adding the supported media (like Image, Map or Video). 
+
+### Images
+
+An image added inside the paragraph section can be customized through the [Content Image Toolbar](content-image-toolbar.md). Below is an example of a small, center-aligned image, just below a text content:
```

Comment:
An image added inside the paragraph section can be customized through the [Image Content Toolbar](content-image-toolbar.md). Below is an example of a small, center-aligned image, just below a text content: